mne/source_space.py

```diff
@@ -3197,3 +3197,110 @@ def _get_src_nn(s, use_cps=True, vertices=None):
     else:
         nn = s['nn'][vertices, :]
     return nn
+
+
+def vertex_depth(inst, info=None, picks=None, trans=None, mode='dist',
```

Comment:
Errant push of `vertex_depth`?

Reply:
Oh no.

Reply:
I never know what to do about the codecov errors. Otherwise I think it's fine now.